readme.md

```diff
@@ -316,6 +316,24 @@ Gets the details for a session.
 * `callback` (`function(error, session)`): A callback to invoke when the API call is complete.
 	* `session`: A [session object](#session-objects).
 
+#### automateClient.updateSession(id, options, callback)
+
+Updates the status of a session.
+
+* `id`: The hashed ID of the session.
+* `options`: An object containing the parameters.
+	* `status`: New status value. [Accepted values](https://www.browserstack.com/automate/rest-api#rest-api-sessions) are `"completed"` or `"error"`.
```

Comment:
For consistency, change

> Accepted values are `"completed"` or `"error"`.

to:

> May be one of `"completed"`, `"error"`.

This may also include `"done"` and `"failed"` based on https://github.com/scottgonzalez/node-browserstack/pull/36#issuecomment-194999038. @shirish87 do you have an ETA for that update?